i2c/lcd_1602_i2c/i2c_lcd.h

```diff
@@ -0,0 +1,26 @@
+/**
+* Copyright (c) 2025 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef I2C_LCD_H
+#define I2C_LCD_H
+#include <stdbool.h>
+#include <hardware/i2c.h>
+
+typedef struct i2c_lcd* i2c_lcd_handle;
+
+i2c_lcd_handle i2c_lcd_init(i2c_inst_t* handle, uint8_t address);
+void i2c_lcd_setCursorLocation(i2c_lcd_handle inst, uint8_t x, uint8_t y);
```

Comment:
I wonder if it'd be better to stick to snake_case (i.e. `i2c_lcd_set_cursor_location`) rather than using a weird mix of snake_case and camelCase?

Reply:
I'm happy to revise the case if desired. This mix is my own convention that I've come to use to help me distinguish function names coming from different "classes"/modules, since C doesn't support namespaces.

Reply:
Fixed